client/network/server-url.ts

```diff
@@ -27,3 +29,17 @@ export function getServerOrigin() {
     return makeServerUrl('')
   }
 }
+
+/**
+ * Returns an URL for a path that is supposed to be in a public assets directory. In case it's not
+ * set (should be very rare), we default to our production server CDN.
+ */
+export function makePublicAssetUrl(path: string) {
+  const serverOrigin = getServerOrigin().toLowerCase()
+  const publicAssetsUrl = new JsonLocalStorageValue<string>(`${serverOrigin}:publicAssetsUrl`)
+  if (publicAssetsUrl) {
```

Comment:
This should handle leading slashes the same way that `makeServerUrl` does (line 17), otherwise this is fraught with issues around whether or not the user provided a leading slash

Reply:
The logic here also doesn't really makes sense, `publicAssetsUrl` is always set. It seems like what you actually wanted was `if (publicAssetsUrl.getValue())`, but you should probably also retrieve that only once to avoid paying for the JSON parse cost twice